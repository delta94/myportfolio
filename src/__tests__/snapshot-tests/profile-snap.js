import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Profile from '../../components/profile';
import ProfileImage from '../mocks/profile-image.json';

it('Profile view renders correctly', () => {
  const wrapper = shallow(<Profile profileImage={{
    sizes: ProfileImage,
  }}
  />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
