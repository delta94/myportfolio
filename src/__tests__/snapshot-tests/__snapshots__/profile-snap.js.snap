// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile view renders correctly 1`] = `
<styled.figure>
  <styled.ul>
    <styled.li>
      Frontend engineer
    </styled.li>
    <styled.li>
      Fullstack developer
    </styled.li>
    <styled.li>
      UI Architect
    </styled.li>
  </styled.ul>
</styled.figure>
`;
